mmv1/products/biglake/Database.yaml

```diff
@@ -20,41 +20,36 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
     "Manage open source metadata with BigLake Metastore": "https://cloud.google.com/bigquery/docs/manage-open-source-metadata#create_databases"
   api: "https://cloud.google.com/bigquery/docs/reference/biglake/rest/v1/projects.locations.catalogs.databases"
-base_url: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases"
-self_link: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases/{{name}}"
-create_url: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases?databaseId={{name}}"
+base_url: "{{catalog}}/databases"
```

Comment:
Does this need to have a `%`? I thought we used % for parameters that can contain slashes

Reply:
Nevermind, that should be in the import format and id format instead of the URL